src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,58 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const currentTab = tabs.find(tab => tab.id === selectedTabId);
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab, index) => {
+            const currentTabSelected = selectedTabId === tab.id;
+            const isActive = currentTabSelected
+            || (!currentTab && !index)
+
+            const changeTab = () => {
+              if (!(currentTabSelected)) {
+                onTabSelected(tab);
+              }
+            }
```

Comment:
oh, it looked much better previous way. Disregard that comment, sorry.
I approve, but this method should be inside the place where it is called